### PR TITLE
add normalizeTests mutation

### LINKED ISSUE #5: [feature]: Mutation to normalise multiple test-cases in a test-run
**Is your feature request related to a problem? Please describe.**
I have to go into every test case detail for a test-run and accept/normalise each. 

**Describe the solution you'd like**
Add a mutation to normalise multiple test-cases(ids) in a test-run

### PR DESCRIPTION
**Pull request by madhavsikka**
- Fixes #5 
- Added `normalizeTests` mutation to normalize multiple test cases at once

### CODE REVIEW

File: graph/schema.graphqls [hunk 1]
```diff
@@ -4,6 +4,7 @@ extend type Mutation {
   # normalizeTest accepts test ID (part of a test run) and updates the respective testcase
   # with the responses from the test result
   normalizeTest(id: String!): Boolean!
```
Comment (slayerjain) [inline graph/schema.graphqls:6]: we should remove the old normalize method, it doesn't make sense to have it anymore.

File: graph/schema.resolvers.go [hunk 2]
```diff
@@ -42,6 +43,20 @@ func (r *mutationResolver) NormalizeTest(ctx context.Context, id string) (bool,
 	return true, nil
 }
 
+func (r *mutationResolver) NormalizeTests(ctx context.Context, ids []string) (bool, error) {
```
Comment (slayerjain) [inline graph/schema.resolvers.go:46]: You should also add a test for this call. I'll add you to the keploy account for this project. There you need to just perform this request, and it should get captured in testmode.

**Review by slayerjain — CHANGES_REQUESTED** [review]

**Comment by slayerjain** [comment]
@madhavsikka as you use keploy while working on keploy - please feel free to share any annoyance or bugs in the process. We'd love to improve the user experience as much as possible :)

File: graph/schema.resolvers.go (hunk 2 above)
Comment (madhavsikka) [reply graph/schema.resolvers.go:46]: Have you added me ?

Comment (slayerjain) [reply graph/schema.resolvers.go:46]: sending you the creds on slack :)

**Comment by madhavsikka** [comment]
- Added test case coverage

**Comment by slayerjain** [comment]
@madhavsikka this is awesome!

**Review by slayerjain — APPROVED** [review]

**Pull request merged**